README.md

```diff
@@ -57,3 +73,9 @@ Tests are written with [mocha](https://mochajs.org)
 ```bash
 npm test
 ```
+
+# Contributors
+
+Fent ([@fent](https://github.com/fent))
+------------------
+* Jesse Spencer ([@Jspenc72](https://github.com/jspenc72))
```

Comment:
I'd rather we use the `contributors` field in `package.json`

https://docs.npmjs.com/files/package.json#people-fields-author-contributors